src/assets/scss/global/fonts.scss

```diff
@@ -5,24 +5,59 @@
 	font-display: swap;
 	src: url("~@/assets/fonts/PostGrotesk-Medium.woff") format("woff");
 }
+
+@font-face {
```

Comment:
We should also remove the old declarations. We only want `woff2` files to be loaded.